### PR TITLE
Implement Login And Signup First Version

### PR DESCRIPTION
**Pull request by HENRYKC24**
## In this pull request, I created the signup and login forms

### CODE REVIEW

File: src/components/pages/SignupPage.js [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+
+  useEffect(() => {
+    setSignUpSucess(() => signedUp);
+  }, [state]);
+
+  return (
+    <form className={cx(colorScheme.green, form)}>
+      <div style={{ backgroundColor: colorScheme.blue }} className={cx(formHeader)}>
+        <h2 className={h2}>Let&apos;s Sign Up</h2>
+        {/* <p className={p}>Fill out this form to sign up with us!</p> */}
```
Comment (Whoistolu) [inline src/components/pages/SignupPage.js:80]: - [ ]  [OPTIONAL] Kindly remove unused lines of code in order to have a clean code-base.

File: src/redux/user/user.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+      userId: user.id,
+      signedUp: true,
+    };
+
+    localStorage.setItem('userAuth', JSON.stringify(authorization));
+
+    // const appState = {
+    //   ...mainUser,
+    // };
+    // delete appState.auth;
```
Comment (Whoistolu) [inline src/redux/user/user.js:113]: - [ ]  [OPTIONAL] Kindly remove unused lines of code in order to have a clean code-base.

File: src/redux/user/user.js [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
```
Comment (Whoistolu) [inline src/redux/user/user.js:161]: - [ ]  [OPTIONAL] Kindly remove unused lines of code in order to have a clean code-base.

File: src/redux/user/user.js [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+      name: '',
+      email: '',
+      loggenIn: 'out',
+      userId: '',
+      signedUp: false,
+    }));
+    console.log('Something went wront');
+  }
+
+  // dispatch(fetchData(data));
```
Comment (Whoistolu) [inline src/redux/user/user.js:175]: - [ ]  [OPTIONAL] Kindly remove unused lines of code in order to have a clean code-base.

**Review by Whoistolu — APPROVED** [review]
### Hi @HENRYKC24 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HENRYKC24** [comment]
I told you this is just for the sake of everybody having the same codebase. Not the final code. Anyways, thanks for the review.

**Comment by Whoistolu** [comment]
> I told you this is just for the sake of everybody having the same codebase. Not the final code. Anyways, thanks for the review.

Hello @HENRYKC24,

This is noted.
Happy coding!!!

**Comment by HENRYKC24** [comment]
:+1:

**Pull request merged**